docs/uml/protocol/ParticipantTriggeredByItself.puml

```diff
@@ -1,6 +1,6 @@
 @startuml
-SimScheduler -> ParticipantAgents: <font color=red>!</font> TriggerWithIdMessage(\n\tIInitializeSystemComponentsTrigger(_),\n\ttriggerId,\n\t_)
-ParticipantAgents --> SimScheduler: <font color=red>!</font> CompletionMessage(triggerId, ScheduleTriggerMessage[...])
-SimScheduler -> ParticipantAgents: <font color=red>!</font> TriggerWithIdMessage(\n\tActivityStartTrigger(tick),\n\t_,\n\t_)
-ParticipantAgents --> SimScheduler: <font color=red>!</font> CompletionMessage(triggerId, None)
+Scheduler -> ParticipantAgents: <font color=red>!</font> TriggerWithIdMessage(\n\tIInitializeSystemComponentsTrigger(_),\n\ttriggerId,\n\t_)
+ParticipantAgents --> Scheduler: <font color=red>!</font> Completion(triggerId, ScheduleActivation[...])
+Scheduler -> ParticipantAgents: <font color=red>!</font> TriggerWithIdMessage(\n\tActivation(tick),\n\t_,\n\t_)
```

Comment:
Please have a look at this :) (referring to our discussion today)

Reply:
```suggestion
Scheduler -> ParticipantAgents: <font color=red>!</font> Activation(INIT_SIM_TICK)
ParticipantAgents --> Scheduler: <font color=red>!</font> Completion(\n\tactor,\n\ttick)
Scheduler -> ParticipantAgents: <font color=red>!</font> Activation(newtick)
ParticipantAgents --> Scheduler: <font color=red>!</font> Completion(\n\tactor,\n\tnewtick+1)
```
Is that how this UML DIagram is supposed to be? I'm not sure about the Actors for Completion and if the ticks are correct.

Reply:
I'd use `Activation(tick)` and `Completion(actor, newTick)` here